src/utils/miscellaneous.ts

```diff
@@ -128,6 +128,13 @@ export async function silentPromise<T>(promise: Promise<T>, customMessage?: stri
   }
 }
 
+export function setStorageItem(key: string, data: unknown): void {
+  // localStorage API accepts only strings
+  // TODO: consider switching to localForage API (accepts all types)
```

Comment:
Maybe remove the TODO and make an issue if u want.
I don't think is a big issue though